web/app.js

```diff
@@ -240,6 +240,7 @@ const PDFViewerApplication = {
   async initialize(appConfig) {
     this.preferences = this.externalServices.createPreferences();
     this.appConfig = appConfig;
+    this._saveInProgress = false;
```

Comment:
Please put the definition after the following line instead: https://github.com/mozilla/pdf.js/blob/7fb01f9f2a4d3d6e2680cf9f6df3c83446d6e763/web/app.js#L237
and also remember to reset it after this line as well: https://github.com/mozilla/pdf.js/blob/7fb01f9f2a4d3d6e2680cf9f6df3c83446d6e763/web/app.js#L733